candi/openapi/doc-ru-cluster_configuration.yaml

```
@@ -60,7 +60,8 @@ apiVersions:
 
           Изменение параметра в работающем кластере приведет к [автоматическому обновлению](https://deckhouse.ru/documentation/v1/modules/040-control-plane-manager/#управление-версиями) версии control plane кластера.
 
-          Если указано `Automatic`, то используется версия control plane, которая считается стабильной на данный момент. Изменение этой версии может произойти при смене минорной версии релиза Deckhouse (см. список изменений к релизу).
+          Если указано `Automatic`, то используется версия control plane, которая считается стабильной на данный момент. Но не меньше одной версии от максимальной версии которая когда либо была использована в кластере.
+          Изменение этой версии может произойти при смене минорной версии релиза Deckhouse (см. список изменений к релизу).
```

```suggestion
          Если указано `Automatic`, то используется версия control plane, которая считается стабильной на данный момент. Выбираемая автоматически стабильная версия также не может отличаться больше чем на одну минорную версию от максимальной версии, которая когда-либо была использована в кластере.
          Изменение этой версии может произойти, например, при смене минорной версии релиза Deckhouse (см. список изменений к релизу).
```
@MariaMakeeva, your turn...